protocol/ethereum_asset_amount.go

```diff
@@ -22,13 +22,23 @@ func GetEthAssetAmount(
 	amt *apd.Decimal, // in standard units
 	asset types.EthAsset,
 ) (coins.EthAssetAmount, error) {
```

Comment:
I'm not sure what to do with this function. It never had direct unit tests and the only caller left is in the `xmrmaker`'s `checkContract` function (different package). It was no longer a good fit in the other 2 places where it was being called, as both of them required the token info well before creating this ETH asset amount.

Reply:
you can move it to the xmrmaker package perhaps?

Reply:
done